Resources/Prototypes/_StarLight/Roles/Jobs/Cargo/mining_specialist.yml

```
@@ -27,3 +27,8 @@
     #storage:
     #back:
     #- Stuff
+
+- type: chameleonOutfit
+  id: MiningSpecialistChameleonOutfit
+  job: MiningSpecialist
+  equipment: {} # may someone tell me what these are supposed to look like when chameleon'd
```

```suggestion
  equipment:
    head: ClothingHeadHatCargosoft
    eyes: ClothingEyesGlassesCheapSunglasses
    mask: ClothingMaskGasExplorer
    outerClothing: ClothingOuterWinterMiner
    neck: ClothingNeckScarfStripedBrown
    gloves: ClothingHandsGlovesColorBlack
```
exactly the same as a salvage